### PR TITLE
Update Helm repo URL

### PR DESCRIPTION
**Pull request by charlychiu**
**Description of your changes:**

- Update docs Helm repo URL

**Which issue is resolved by this Pull Request:**
Resolves #

### CODE REVIEW

**Comment by yangkyeongmo** [comment]
I had same issue caused by this, but just appending `stable` to existing url fixed the issue.

**Comment by zimnx** [comment]
Thanks, please amend PR and commit title in a way it would explain what is changed as a sentence. Check commit history for a style reference.
/ok-to-test
/kind documentation

**Comment by charlychiu** [comment]
> zimnx

Corrected

**Comment by charlychiu** [comment]
@zimnx 

Corrected, thx your reviewed

File: helm/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
 Once Helm is set up properly, add the repo as follows:
 
 ```console
-helm repo add scylla-operator https://storage.googleapis.com/scylla-operator-charts
+helm repo add scylla https://scylla-operator-charts.storage.googleapis.com/stable
```
Comment (zimnx) [inline helm/README.md:19]: please keep repo name as it was because next commands refer to it.

Comment (charlychiu) [reply helm/README.md]: Corrected

**Review by zimnx — APPROVED** [review]
Thanks for contribution and updates!
/lgtm

**Comment by zimnx** [comment]
CI is borked today, it will get merged once we fix it.

**Comment by rzetelskik** [comment]
> @charlychiu: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> ci/prow/e2e-gke-parallel-clusterip	[533914a](https://github.com/scylladb/scylla-operator/commit/533914a5e5a3d80d31f1538adcae42e4a763eeb1)	[link](https://prow.scylla-operator.scylladb.com/view/gs/scylla-operator-prow/pr-logs/pull/scylladb_scylla-operator/1887/pull-scylla-operator-master-e2e-gke-parallel-clusterip/1785316039039389696)	unknown	`/test e2e-gke-parallel-clusterip`
> [Full PR test history](https://prow.scylla-operator.scylladb.com/pr-history?org=scylladb&repo=scylla-operator&pr=1887). [Your PR dashboard](https://prow.scylla-operator.scylladb.com/pr?query=is:pr+state:open+author:charlychiu).

known manager flake, unrelated
/retest

**Pull request merged**